htdocs/core/class/extrafields.class.php

```diff
@@ -827,26 +827,26 @@ function showInputField($key, $value, $moreparam='', $keyprefix='', $keysuffix='
 			if (! is_object($form)) $form=new Form($this->db);
 
 			// TODO Must also support $moreparam
-			$out = $form->select_date($value, $keysuffix.'options_'.$key.$keyprefix, $showtime, $showtime, $required, '', 1, 1, 1, 0, 1);
+			$out = $form->select_date($value, $keyprefix.'options_'.$key.$keysuffix, $showtime, $showtime, $required, '', 1, 1, 1, 0, 1);
```

Comment:
Variable at same place in function signature are not used for same usage. This surely break compatibility. Are you sure of this ?

Reply:
I can't answer about because i don't try with. t was a clear code for me and i changed every keyword keysuffix and keyprefix in this file only. I'm sure in this file about working process but i don't know about other files